apps/src/netsim/NetSimMetronome.js

```diff
@@ -72,6 +72,13 @@ NetSimMetronome.prototype.tick = function (clock) {
     this.lastPulseTime_ = clock.time;
   }
 
+  // An infinite interval means we're effectively paused, so snap to "empty"
```

Comment:
Good catch, that's not very descriptive.  Maybe it should say "snap to zero progress."  It looks like this:
![empty_metronome](https://cloud.githubusercontent.com/assets/1615761/7241222/331ce342-e76c-11e4-9560-ab5dcdad4a5c.png)